app/models/analytics_data_layer.rb

```diff
@@ -31,7 +31,12 @@ def add(data = {})
   def add_user_info(user)
     @analytics_data[:userType] = user.user_description if user
     @analytics_data[:userRegisterAndPartnerId] = user.register_and_partner_id if user
-    @analytics_data[:userCoreInductionProgramme] = user.core_induction_programme if user
+    @analytics_data[:userCoreInductionProgramme] = add_cip_info(user.core_induction_programme) if user
+    @analytics_data[:cohortStartYear] = add_cohort_start_year(user) if user&.participant?
+  end
+
+  def add_cohort_start_year(user)
+    user.cohort&.start_year
```

Comment:
That does not look right 😂 

`add_xxx` are methods that do `@analytics_data[:yyy] = zzz`

I would do
```
    # in add_user_info
    @analytics_data[:cohortStartYear] = user.cohort.start_year if user&.participant?
```
or
```
    # in add_user_info
    add_cohort_start_year(user) if user&.participant?

  def add_cohort_start_year(user)
    @analytics_data[:cohortStartYear] = user.cohort.start_year if user&.participant?
  end
```

Reply:
My bad, yep, i'll go with the former.